pkg/cluster_upgrader/cluster_upgrader_verification_test.go

```diff
@@ -0,0 +1,324 @@
+package cluster_upgrader
+
+import (
+	"fmt"
+	"github.com/go-logr/logr"
+	"github.com/golang/mock/gomock"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	configv1 "github.com/openshift/api/config/v1"
+	upgradev1alpha1 "github.com/openshift/managed-upgrade-operator/pkg/apis/upgrade/v1alpha1"
+	"github.com/openshift/managed-upgrade-operator/pkg/metrics"
+	testStructs "github.com/openshift/managed-upgrade-operator/util/mocks/structs"
+	appsv1 "k8s.io/api/apps/v1"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+	logf "sigs.k8s.io/controller-runtime/pkg/log"
+
+	mockMaintenance "github.com/openshift/managed-upgrade-operator/pkg/maintenance/mocks"
+	mockMetrics "github.com/openshift/managed-upgrade-operator/pkg/metrics/mocks"
+	"github.com/openshift/managed-upgrade-operator/util/mocks"
```

Comment:
We usually split the imports into different groups

Reply:
Thanks for the tip! I've structured them into more sensible groups now.